### DISCUSSION TITLE
UI improvements

### DISCUSSION THREAD

**Discussion by alphabetek** [AllYourBot/hostedgpt#477] [General]
I've noticed small UI improvements that could be made:

**Overflow tooltip issue**
In Hostedgpt
<img width="783" alt="Screenshot 2024-07-29 at 18 45 16" src="https://github.com/user-attachments/assets/463ccd21-464d-4062-a580-6dbb113551d6">
Expected behavior (ChatGPT like)
<img width="788" alt="Screenshot 2024-07-29 at 18 46 49" src="https://github.com/user-attachments/assets/6d4aaa36-2fc5-46d6-965e-bde8fc4a3e3a">

**Avatar overflow issue (if the user has a long name)**
In Hostedgpt
<img width="214" alt="Screenshot 2024-07-29 at 18 49 19" src="https://github.com/user-attachments/assets/33ea2142-1dda-4d55-8160-44f7173bb869">
Expected behavior
Maybe like in ChatGPT, there is no need for users name initials (or take only two letters) + make the size of the model avatar the same as the user one (circle, not elliptic).

**Dropdown menu component click**
In Hostedgpt (everywhere where this component is used)
<img width="263" alt="Screenshot 2024-07-29 at 18 52 25" src="https://github.com/user-attachments/assets/3fd39868-5b08-4d3b-b6e8-f5877d7c1bd4">
Expected behavior
On click, dropdown menu should open (like now), if clicked again it should be closed (right now the user has to click outside the component to close the menu).

**Comment by keithschacht** [comment]
Thanks for the write-up and screenshots. This is helpful.

On the avatar name issue it definitely should be just grabbing the initials. It does that on my account. Can you share with me what the text of the full name field is? That will help me to debug why it's not truncating properly.

**Reply by alphabetek** [reply]
Yes, just try a composable first name or last name, like "AA" as first name and "CC DD EE" as last name, it's common to have spaces in names, so maybe a trim on names could fix the issue.
